latexify/core.py

```diff
@@ -163,9 +167,44 @@ def visit_Compare(self, node):
 
     if isinstance(node.ops[0], ast.Eq):
       return lstr + '=' + rstr
+    # CHANGE:(20200801@1MLightyears) Add support for conditions for <,>,\ge,\le,\ne,is
+    elif isinstance(node.ops[0], ast.Gt):
+      return lstr + '>' + rstr
+    elif isinstance(node.ops[0], ast.Lt):
+      return lstr + '<' + rstr
+    elif isinstance(node.ops[0], ast.GtE):
+      return lstr + r'\ge ' + rstr
+    elif isinstance(node.ops[0], ast.LtE):
+      return lstr + r'\le ' + rstr
+    elif isinstance(node.ops[0], ast.NotEq):
+      return lstr + r'\ne ' + rstr
+    elif isinstance(node.ops[0], ast.Is):
+      return lstr + r'\space is\space ' + rstr
+    elif isinstance(node.ops[0], ast.IsNot):
+      return lstr + r'\space is\space not\space ' + rstr
+
     else:
       return r'\operatorname{unknown\_comparator}(' + lstr + ', ' + rstr + ')'
 
+  # CHANGE:(20200801@1MLightyears) Add support for BoolOp, which allow Latexify to
+  # deal with more complex boolean statements.
+  # The structure is weak, for I seldom use very complex boolean statements and
+  # currently have no idea how to simply implement a result like "x=0,1" instead of
+  # "x=0 Or x=1" as it is now XD
+  def visit_BoolOp(self, node):
+    '''
+    When node is an instance of ast.BoolOp, it has:
```

Comment:
This information is already provided by `ast`. We need not to provide the same stuff.

Reply:
Forget to delete one of the ''' comments - will be delete in the next commit, sorry!